docs/api/amqp.md

```diff
@@ -172,6 +173,44 @@ JSON in the following format:
 }
 ```
 
+### Schema status
+
+Message with the status of schema update command.
+
+#### Exchange
+
+* `fog`
+
+#### Binding Key
+
+* `schema.updated`
+
+#### Expected Response
+
+JSON in the following format:
+  * `id` **String** device's ID
+  * `updated` **Boolean** updated result.
+  * `error` **String** (optional) a string with detailed error message.
+
+#### Success Response Example
+
+```json
+{
+  "id": "3aa21010cda96fe9",
+  "updated": true
+}
+```
+
+#### Error Response Example
+
+```json
+{
+  "id": "3aa21010cda96fe9",
+  "updated": false,
```

Comment:
Is there any chance of `updated` being `false` and no error occur? I think you can return just `id` and `error` properties.

Reply:
I agree with you guys. I will change to return just the error. If the message was successfull the error is `null`

Reply:
@lcbm good point!
I agree with you that we must define a standard API, considering the mentioned criteria. However, I don't think the specified protocol fields are suitable for handling application errors since they were not designed to them (just broker <-> library). The reason is that one of the principles behind the AMQP protocol is to let the developers do what they want (what matters is sending bytes to one side from another).

Let's talk about it later this week.